### PR TITLE
getting started reach

### PR DESCRIPTION
**Pull request by rfustino**

### CODE REVIEW

**Review by JasonWeathersby — APPROVED** [review]

**Review by ryanRfox — COMMENTED** [review]
I feel index.md should also be modified to direct readers toward PyTeal and/or Reach. Currently there is a link to docs.reach.sh and no link to PyTeal in the final sections.

File: docs/get-started/dapps/reach.md [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+
+Set an environment variable to use the Algorand Blockchain. 
+
+
+```
+export REACH_CONNECTOR_MODE="ALGO-devnet"
+```
+
+
+# Application Overview
```
Comment (ryanRfox) [inline docs/get-started/dapps/reach.md:121]: I like the overview section from PyTeal, as it has some visuals.

**Review by ryanRfox — CHANGES_REQUESTED** [review]
Trim some bits and move a section around.

File: docs/get-started/dapps/reach.md [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+``` bash
+cd reach-auction
+```
+
+Next, download Reach by running
+``` bash
+$ curl https://docs.reach.sh/reach -o reach ; chmod +x reach
+```
+
+Confirm the download worked by running
```
[the remark this answers is not in the document]
Comment (rfustino) [reply docs/get-started/dapps/reach.md]: I updated index.md w link to reach at bottom as well, in commints: https://github.com/algorand/docs/pull/674/commits/36882f3230f12b0d97754c12d28fd25e1df6bfd0 and https://github.com/rfustino/docs/tree/3d66e5921a2662c0d901086acbca232df9d434b7 . Removed confirm download and using ./reach version .

File: docs/get-started/dapps/reach.md [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+
+
+Since Reach is Dockerized, when first used, the images it uses need to be downloaded. This will happen automatically when used for the first time, but can be done manually now by running
+
+
+```
+  $ ./reach update
+```
+
+
```
[the remark this answers is not in the document]
Comment (rfustino) [reply docs/get-started/dapps/reach.md:70]: removed compile --help. and hashes

File: docs/get-started/dapps/reach.md [hunk 4]
```diff
@@ -0,0 +1,474 @@
...
+```
+export REACH_CONNECTOR_MODE="ALGO-devnet"
+```
+
+
+# Application Overview
+
+
+## Alice and Bob’s auction design
```
[the remark this answers is not in the document]
Comment (rfustino) [reply docs/get-started/dapps/reach.md:96]: Good idea, made a reference.

File: docs/get-started/dapps/reach.md [hunk 5]
```diff
@@ -0,0 +1,474 @@
...
+
+
+
+1. Sellers must be able to create a new auction for each piece of artwork. The artwork must be held by the contract after the auction begins and until the auction closes.
+2. The auction can be closed before it begins, in which case the artwork will be returned to the seller.
+3. Sellers can specify a reserve price, which if not met, will return the artwork to them.
+4. For each bid, if the new bid is higher than the previous bid, the previous bid will be refunded to the previous bidder and the new bid will be recorded and held by the contract.
+5. At the end of a successful auction, where the reserve price was met, the highest bidder will receive the artwork and the seller will receive the full bid amount.
+
+## Reach architecture
```
[the remark this answers is not in the document]
Comment (rfustino) [reply docs/get-started/dapps/reach.md]: Moved to top, added more why/what vs PyTeal. Visual architecture will be in detail doc.

File: docs/get-started/dapps/reach.md (hunk 1 above)
Comment (rfustino) [reply docs/get-started/dapps/reach.md:93]: added a visual

**Review by ryanRfox — APPROVED** [review]
lgtm

**Pull request merged**